src/main/java/bio/terra/workspace/service/iam/model/ControlledResourceIamRole.java

```diff
@@ -28,7 +28,7 @@
   public static ControlledResourceIamRole fromApiModel(ApiControlledResourceIamRole apiModel) {
```

Comment:
please add unit tests for this function.
What's the correct return value is if the argument `apiModel` is null?

Reply:
I think it's reasonable to throw in that case, added a ValidationException here